wordpress-mobile/gutenberg-mobile
# Fix crashes editing More block

This PR fixes an issue on the More block, that makes the app unstable/crashing when editing the content of the block. Fix #316 

GB side PR here: https://github.com/WordPress/gutenberg/pull/12620

**Testing Steps**
- Open the demo app (android or iOS)
- Tap into the "Read more" block. 
- Edit the content of the block

It should not crash.

Test the same on the `Code` block, that internally uses `PlainText`.
